e2e/dractions/retry.go

```diff
@@ -15,38 +15,28 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-// nolint:gocognit
-// return placementDecisionName, error
-func waitPlacementDecision(client client.Client, namespace string, placementName string,
-) (string, error) {
+// waitPlacementDecsision waits until we have placemnt desicion and return the placmenet decision object.
+func waitPlacmentDecision(client client.Client, namespace string, placementName string,
```

Comment:
Typo: waitPlacementDecision

Reply:
I think I fixed all the typos, thanks for reviewing!